ReactCommon/react/renderer/components/legacyviewmanagerinterop/RCTLegacyViewManagerInteropCoordinator.h

```diff
@@ -20,13 +20,13 @@ typedef void (^InterceptorBlock)(std::string eventName, folly::dynamic event);
 
 - (instancetype)initWithComponentData:(RCTComponentData *)componentData bridge:(RCTBridge *)bridge;
 
-- (RCTUIView *)createPaperViewWithTag:(NSInteger)tag;
+- (RCTUIView *)createPaperViewWithTag:(NSInteger)tag; // TODO(macOS GH#774)
 
 - (void)addObserveForTag:(NSInteger)tag usingBlock:(InterceptorBlock)block;
 
 - (void)removeObserveForTag:(NSInteger)tag;
 
-- (void)setProps:(folly::dynamic const &)props forView:(RCTUIView *)view;
+- (void)setProps:(folly::dynamic const &)props forView:(RCTUIView *)view; // TODO(macOS GH#774)
```

Comment:
This file lives in ReactCommon, which I thought was platform agnostic C++ code? I'm surprised to see UIKit references here?

Reply:
@christophpurrer mentioned that in theory there shouldn't be any platform specific code in this layer, but in practice the abstraction leaks a bit.

Reply:
Future thinking: I wonder if this is an area where we could either:

1) Move this file to the iOS Fabric path
2) Introduce an... abstract type for the view type so we don't directly reference UIKit. Almost like... a shim layer 😅 

I don't consider this comment blocking. Might be interesting for the future though.